config/products.ts

```diff
@@ -1,5 +1,20 @@
-import capitalize from '@hashicorp/platform-util/text/capitalize'
-import { ProductGroup, ProductSlug } from 'types/products'
+import { ProductGroup, ProductName, ProductSlug } from 'types/products'
+
+/**
+ * A map of product slugs to their proper noun names.
+ */
+const productSlugsToNames: { [slug in ProductSlug]: ProductName } = {
+  boundary: 'Boundary',
+  consul: 'Consul',
+  hcp: 'HashiCorp Cloud Platform',
+  nomad: 'Nomad',
+  packer: 'Packer',
+  sentinel: 'Sentinel',
+  terraform: 'Terraform',
+  vagrant: 'Vagrant',
+  vault: 'Vault',
+  waypoint: 'Waypoint',
+}
```

Comment:
Adding this removes the need for `@hashicorp/platform-util/text/capitalize` and also fixes a TS error that appeared after I added the `ProductName` type in `src/types/products.ts`.